lang/setup_ar.lang

```diff
@@ -647,3 +647,5 @@ cache_activated_components#:#Activated Components###30 08 2015 new variable
 cache_flush#:#Flush all caches###30 08 2015 new variable
 cache_activate_tpl_blocks#:#Template (Blocks)###30 08 2015 new variable
 cache_activate_tpl_variables#:#Template (Variablen###30 08 2015 new variable
+error_log_path#:#Path to error log files
```

Comment:
Nur zur Info: du brauchst nur die englische und evtl. die deutsche Srachdatei pflegen. Die Einträge in andere Sprachdateien fürhrt regelmäßig ein Skript durch, das neue Einträge mit (###30 08 2015 new variable)-Kommentaren versieht.